packages/component-base/test/slot-controller.test.js

```diff
@@ -305,4 +305,37 @@ describe('slot-controller', () => {
       expect(teardownSpy.called).to.be.false;
     });
   });
+
+  describe('multiple nodes', () => {
```

Comment:
**thought:** It would be good to test observing logic with the new parameter too, but it isn't a blocker.

Reply:
For now it's not going to be used, because this is also how `vaadin-confirm-dialog` used to work previously.
See #4998 where I'm using `observe: false` in the controller. But I agree, we should fix that in the future.

Reply:
I agree, we can neglect that. However, I would still suggest adding a test verifying that the default node is properly created when no custom one is provided.

Reply:
Thanks, added missing tests.